content/en/security/threats/workload_security_rules/_index.md

```diff
@@ -19,9 +19,68 @@ further_reading:
   text: "Learn more about Security notification variables"
 ---
 
-With Cloud Security Management Threats (CSM Threats) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of out-of-the-box rules to detect suspicious behavior. CSM Threats rules consist of two different components: [Agent rules](#agent-rules) and [detection rules](#detection-rules).
+This topic explains how Cloud Security Management Threats (CSM Threats) actively monitors system activity and evaluates it against a set of out-of-the-box (OOTB) rules to detect suspicious behavior.
 
-## Agent rules
+## CSM Threats rules construction
+
+CSM Threats rules consist of two different components: Agent rules and Threat detection rules.
```

Comment:
My suggested changes include lowercasing "threat detection rules" and using only "Agent rules" instead of "Agent Configuration rules". These are partly based on personal preference. But as a general rule of thumb, Agent (when referring to the Datadog Agent) is always capitalized. For threat detection rules, the words are capitalized in the UI, but I'm not entirely convinced it's a proper noun. 

Similarly, I know that the UI has an "Agent Configuration" page, but I think the "Configuration" part is more in reference to what you do on the page vs. it being the name for this particular type of rule (Agent rule). You may have already discussed this with the PM, in which case, you can disregard these particular suggestions if they conflict with what they told you 😅.